clio-server/src/test/scala/org/broadinstitute/clio/server/TestKitSuite.scala

```diff
@@ -11,9 +12,12 @@ import org.scalatest.{BeforeAndAfterAll, Suite}
   */
 abstract class TestKitSuite(actorSystemName: String)
     extends TestKit(ActorSystem(actorSystemName))
-    with Suite
+    with AsyncFlatSpecLike
+    with Matchers
     with BeforeAndAfterAll {
 
+  protected implicit val materializer: Materializer = ActorMaterializer()
```

Comment:
Please add a comment describing why this is necessary.

Reply:
https://www.superloopy.io/articles/2013/scalatest-with-akka.html

Looks like collecting traits common to our use of TestKit and Scalatest in one place.

Reply:
That linked code doesn't use `ActorMaterializer()`. Maybe if I understood more about how the testing code worked this would be obvious to me.